### PR TITLE
[10.0][MIG] website_menu_multilang: Migration to 10.0

### PR DESCRIPTION
**Pull request by reinka**
I ran all the commands specified in the migration guide however there wasn't any 9.0 commits.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: website_menu_multilang/model/website.py [hunk 1]
```diff
@@ -3,40 +3,32 @@
...
 
-    _defaults = {
-        'lang_ids': _default_lang_ids,
-    }
+    lang_ids = fields.Many2many(
+        'res.lang', 'website_menu_lang_rel', 'menu_id', 'lang_id',
+        'Allowed Languages',
+        help="Website languages this menu should be available for. "
+             "If kept empty, it will be available for all languages",
+        default=_default_lang_ids
```
Comment (yajo) [inline website_menu_multilang/model/website.py]: The common pattern is to declare fields on top, methods next.

Here you need then to use a `lambda self: self._default_lang_ids()` to be able to do that (which, in turn, is another requirement).

File: website_menu_multilang/view/website.xml [hunk 2]
```diff
@@ -1,26 +1,23 @@
...
+    <field name="inherit_id" ref="website.menu_tree"/>
+    <field name="arch" type="xml">
+      <field name="parent_id" position="after">
+        <field name="lang_ids" widget="many2many_tags"/>
+      </field>
+    </field>
+  </record>
 
-	<!-- website.menu -->
-        <record id="menu_tree_inherit" model="ir.ui.view">
```
Comment (yajo) [inline website_menu_multilang/view/website.xml]: We all have likings about indentation, but please keep a smaller diff by respecting author's decision 😊

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by reinka** [comment]
@Yajo Done.

**Review by yajo — APPROVED** [review]
Reminder: squash at merge.

File: website_menu_multilang/__manifest__.py [hunk 3]
```diff
@@ -24,7 +24,7 @@
 
 {
     'name': 'Website Menu Multilanguage',
-    'version': '8.0.0.1.0',
+    'version': '10.0.0.1.0',
     'author': 'Versada UAB,Odoo Community Association (OCA)',
     'category': 'Website',
     'website': 'http://www.versada.lt',
```
Comment (simahawk) [inline website_menu_multilang/__manifest__.py:30]: https://github.com/OCA/website

**Review by simahawk — CHANGES_REQUESTED** [review]

File: website_menu_multilang/__manifest__.py (hunk 3 above)
Comment (simahawk) [reply website_menu_multilang/__manifest__.py:30]: @reinka  can you fix this and squash on push? Then we can merge, tnx

**Pull request closed without merging**